### DISCUSSION TITLE
2023-05-03 Meeting Agenda, Recording, and Notes

### DISCUSSION THREAD

**Discussion by rfay** [ddev/ddev#4857] [DDEV Advisory Group]
Previous meeting: 
* #4517
Next meeting: 5 July 2023

## Agenda

Agenda:
* Turn on recording, ask for a minutes-taker
* ddev.com has been converted from WordPress to a lovely Astro-based static website, thanks @mattstein !
* All things DDEV
  * Moved to github.com/ddev (DDEV org or GitHub)
  * Moved docker images to "ddev" org on hub.docker.com
  * Paperwork/trademark license almost complete to allow changing Localdev Foundation to DDEV Foundation (actually doing the change is still ahead)
* Long-term sustainability: Finding paid co-maintainers, finding support to pay them.
* Concerns: Availability of Simon Gillis and Matt Stein
* DDEV v1.21.5/6 were released with new features and bugfixes
* DDEV v1.22.0 upcoming, with add-on versioning+removal+upgrade, Python/Django support, etc. Node.JS defaults and PHP defaults changing. Traefik will becom default router, pre-start execution.
* 501(c)(3) application for Localdev/DDEV Foundation
* Randy will be attending Drupalcon NA and Drupalcon EU, and Craft CMS Dot All 2023. Riding bike from DC to Pittsburgh for Drupalcon NA. Possibly attending TYPO3 Developer Days in Karlsruhe.
* Financial status/reports
* Anything on anybody's mind!

## Meeting Notes

https://docs.google.com/document/d/1ou5aHUwvguKuuj6-z1qIoV5-Qm8oMb2cNDHNsxv_-zE/edit

## Recording

[Recording](https://www.dropbox.com/s/7znqrryspfi5jcy/2023-05-03%20DDEV%20Advisory%20Group.mp4?dl=0)

## Financial reports

* Localdev Foundation had to pay a total of $3115 of Federal and State Income Taxes :(
[Localdev Foundation Balance Sheet 2023-04-30-16_25.pdf](https://github.com/ddev/ddev/files/11361772/Localdev.Foundation.Balance.Sheet.2023-04-30-16_25.pdf)
[Localdev Foundation Profit and Loss 2023-04-30-16_23.pdf](https://github.com/ddev/ddev/files/11361773/Localdev.Foundation.Profit.and.Loss.2023-04-30-16_23.pdf)

**Comment by chrisfromredfin** [comment]
Meeting notes are here!
https://docs.google.com/document/d/1ou5aHUwvguKuuj6-z1qIoV5-Qm8oMb2cNDHNsxv_-zE/edit
